### ISSUE TITLE
Microsoft Login Issue.

### ISSUE THREAD

**Issue by xISHRAKx** [UltimMC/Launcher#357]
### Operating System

Windows

### Description of bug

While trying to Login through Microsoft Login, I get prompted with a Tab to use the token in microsoft to login, After doing that, It processed and gives out This error.
`Failed to get Minecraft access token: Error transferring https://api.minecraftservices.com/launcher/login - server replied: Forbidden
`

So I tried to do it with the Official MultiMC version, And it logged in fine.

### Steps to reproduce

• In 0.7.0 Update Go to Accounts and Select "Add Microsoft"
• Open the link and enter the Code.
• Proceed and in the UltimMC, The progress bar will go up but at the end you will meet with an Error.

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by Darthagnon** [comment]
There is a hard-coded Microsoft API key in https://github.com/UltimMC/Launcher/blob/develop/secrets/Secrets.cpp - maybe it has been revoked/banned for UltimMC? 

It would be better if it was not hard-coded and we could supply our own API keys in settings.

**Comment by xISHRAKx** [comment]
> There is a hard-coded Microsoft API key in https://github.com/UltimMC/Launcher/blob/develop/secrets/Secrets.cpp - maybe it has been revoked/banned for UltimMC?
> 
> It would be better if it was not hard-coded and we could supply our own API keys in settings.

Will there be any patches for this?
I want to use ultimMC for quick alt and stuff for offline server but also want to use the launcher for my actual account....

**Comment by Darthagnon** [comment]
At the moment, development is more or less dormant. If I knew enough, I would make the change myself. EDIT: [PollyMC](https://github.com/fn2006/PollyMC) has already fixed this issue - Settings >> APIs >> API Keys replaces Settings >> Log Upload

If you're on Win7 or higher, I recommend PollyMC. UltimMC is only better because it is the original libre Minecraft launcher fork, and it is built in Qt 5.6.3, which means it works all the way back to Windows XP (tested it myself).

**Comment by TheShadowGamer06** [comment]
Yeah I also just discovered this issue, and very much would love to use the same launcher for everything and not have to use multiple because of some stupid bs. Ultim also has a similar setting you mentioned but im not sure if thats the same Log upload???

**Comment by TheShadowGamer06** [comment]
they changed the key two weeks ago roughly and I still get the same forbidden error, this launcher is just cooked
